spec.txt

```diff
@@ -1093,7 +1093,7 @@ closing sequence of any number of unescaped `#` characters.
 The opening sequence of `#` characters must be followed by spaces or tabs, or
 by the end of line. The optional closing sequence of `#`s must be preceded by
 spaces or tabs and may be followed by spaces or tabs only.  The opening
-`#` character may be indented 0-3 spaces of indentation.  The raw contents of
+`#` character may be preceded by 1-3 spaces of indentation.  The raw contents of
```

Comment:
Sometimes you say "preceded by 1-3 spaces of indentation" and sometimes "preceded by [at most|no more than|up to] three spaces of indentation."  For what it's worth, I like the latter formulations better.  And using both raises the possibility that some readers will try to discern a distinction between them. What do you think?

Reply:
Nice. I don’t remember if I did that or whether it was already so. Anyway, I looked through the doc again and made the language regarding (optional) indentation more clear: https://github.com/commonmark/commonmark-spec/pull/618/commits/69abb8164061d12ea3f22f094b1ed8ee4f882c3d

Reply:
I picked `up to` as its short, and felt the most natural to me. Something else can also work.